packages/slate-editor/src/extensions/floating-add-menu/FloatingAddMenuExtension.tsx

```diff
@@ -1,12 +1,11 @@
 import type { Extension } from '@prezly/slate-commons';
-import { isHotkey } from 'is-hotkey';
-import type { KeyboardEvent } from 'react';
+import { isHotkey, type KeyboardEventLike } from 'is-hotkey';
 
 import { isMenuHotkey, MENU_TRIGGER_CHARACTERS, shouldShowMenuButton } from './lib';
 
-function isTriggerInput(event: KeyboardEvent) {
-    return MENU_TRIGGER_CHARACTERS.some((tiggerKey) =>
-        isHotkey(tiggerKey, { byKey: true }, event.nativeEvent),
+function isTriggerInput(event: KeyboardEventLike) {
+    return MENU_TRIGGER_CHARACTERS.some((triggerKey) =>
+        isHotkey(`shift?+${triggerKey}`, { byKey: true })(event),
```

Comment:
In that case probably better to save `isHotkey(`shift?+${triggerKey}`, { byKey: true })` in order not to create the function every time

Reply:
I have reworked the hotkey detection because it was using two identical characters to make the check. I've noticed that @OskarGasiorowski fixed it in https://github.com/prezly/slate/pull/146 but he also enabled the `byKey` option which actually fixes the issue. Adding the "second" trigger character was completely useless since it was the same character anyway 😄 

From the package description itself
> By default the hotkey string is checked using event.which. But you can also pass in byKey: true to compare using the [KeyboardEvent.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) API, which stays the same regardless of keyboard layout.

So, when using `byKey` it doesn't matter if you press the regular slash on US keyboard, shift+something on non-US keyboard or numpad divide. `event.key` will always be `"/"`.

I've also defined the functions inside the module directly so it won't be recreated on every call.

See https://github.com/prezly/slate/pull/352/commits/25e89ae5f6ee7da8f3754aa6b93d629778ff1472.

cc @e1himself @aspirisen